xbmc/cores/VideoPlayer/Interface/StreamInfo.h

```diff
@@ -64,6 +74,7 @@ struct VideoStreamInfo : StreamInfo
   CRect VideoRect;
   std::string stereoMode;
   int angles = 0;
+  StreamHdrType m_iHdr_Type = StreamHdrType::HDR_TYPE_NONE;
```

Comment:
```suggestion
  StreamHdrType hdrType = StreamHdrType::HDR_TYPE_NONE;
```

Reply:
or `HdrType`

Reply:
Thanks for the review!  Went with your first suggestion here as it's consistent with what I had written in other areas.  Also simplified the detection as per your other comment.